peregrine/iqgen/bits/amplitude_factory.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2016 Swift Navigation Inc.
+#
```

Comment:
I think we have to put the contact info here llke this:
# Contact: Adel Mamin <adelm@exafore.com>